File: src/main/java/com/ttknp/springbootondockerapp/entity/Romance.java
package com.ttknp.springbootondockerapp.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@AllArgsConstructor
@Data
@Entity
@Table(name = "romances")
public class Romance {
    @Id
    private String rid;
    private String title;
    private Integer price;
}

File: src/app/entity/romance.ts

export class Romance {
  public id : number
  public title : string
  public description :string
  public price : number

  constructor(id: number, title: string, description: string, price: number) {
    this.id = id;
    this.title = title;
    this.description = description;
    this.price = price;
  }
}
